src/Api/Endpoints/Apps.cs

```diff
@@ -127,6 +116,12 @@ public static void MapAccountEndpoints(this WebApplication app)
             .RequireCors("default");
     }
 
+    public static async Task<IResult> IsAppIdAvailableAsync([AsParameters] GetAppIdAvailabilityRequest payload, ISharedManagementService accountService)
```

Comment:
Personally prefer the method to be inlined as it was before

Reply:
No